README.md

```diff
@@ -1,87 +1,94 @@
+In its guts (or heart?), Archipelago does something quite simple but core to our concept of repository: it transforms in realtime the _close to your needs open schema metadata_ that lives in strawberryfield as JSON into _close to other one's fixed schema needs metadata_; any destination format, using a fast, cached templating system. A templating system easy enough to master and core to Drupal, called [`Twig`](https://twig.symfony.com/doc/2.x/).
 
-This templating system is exposed to Archipelago users through the UI and stored side by side in the repository as content 
+This templating system is exposed to Archipelago users through the UI and stored side by side in the repository as content
 (we named them `Metadata Display entities`, but they not only serve display needs!) so users can fully control how metadata is transformed and published without touching their individual sources.
 
 Templates or recipes can be shared, exported, ingested, updated, and adapted in many ways. Fast changes are possible without having to wait for the next mayor release of Archipelago or your favorited Metadata Schema Specs Committee agreeing on the next or the last version. Of course, this module not only handles metadata but media assets too, extracting local or remote URIs and files from your metadata and rendering them as media viewers: books, 3D models, images, panoramas, A/V with IIIF in its soul. Discover what [`format_strawberryfield can`](format.md) do and what many other possibilities are exposed through our templating system.
 
-# Archipelago Deployment Quickstart
-
-To get a general glance of what is included and how it works deploy our latest beta locally using `Docker` using [this Guide](https://github.com/esmero/archipelago-deployment/blob/8.x-1.0-beta1/README.md) covering for now OSX and Ubuntu 18.04.
+# Code of Conduct
+The Archipelago Commons community is dedicated to providing a welcoming and positive experience for all participants. To ensure this can be met for all parties, please review our [Code of Conduct](CODE_OF_CONDUCT.md).
 
-# Archipelago Usage Quickstart
+# Archipelago Deployment Quickstart
+To get a general glance of what is included and how it works, follow [this guide](https://github.com/esmero/archipelago-deployment/blob/8.x-1.0-beta2/README.md) to deploy our latest beta locally using `Docker` with OSX or Ubuntu 18.04.
 
-Now that you have it running, it's time to ingest your first digital object. Follow this [Guide](docs/firstobject.md) to master the basics.
+# Archipelago Usage Guides
+Now that you have Archipelago running, the following guides will help you master the basics.
+* [Ingesting Your First Object](docs/firstobject.md)
+* [Creating Display Modes](docs/createdisplaymodes.md)
+* [How to Create a Webform as an Input Method for Archipelago Digital Objects (ADO)](docs/webformsasinput.md)
+* [Debugging PHP in Archipelago](docs/xdebug.md)
 
 # Contributing
-
```

Comment:
There is inconsistency here. Or you remove all spaces after a title, or you keep them. I kinda like them, but in any case we need to be consistent. Up to you what road you follow.